src/api.jl

```diff
@@ -134,12 +134,12 @@ function infer(ra_range::Tuple{Float64, Float64},
     # Get indicies of entries in the  RA/Dec range of interest.
     entry_in_range = entry->((ra_range[1] < entry.pos[1] < ra_range[2]) &&
```

Comment:
Am I missing something, or is this still going to fail to find sources immediately outside the RA, DEC rectangle, but which influence the rectangle?  We talked about adding a border.  When will we do that?  Is the plan to do the run without it?